spec/classes/document_search_spec.rb

```diff
@@ -209,6 +209,24 @@
       end
     end
 
+
+    %w[doc docx pdf ppt xls xlsx].each do |ext|
+      context 'when the results contain demoted and non-demoted file types' do
+        let(:query) { 'rutabaga' }
```

Comment:
To clean up this spec, I would just use the standard `let` variables, because there's no reason to use a custom query for this.

Reply:
Thanks. This is much better.